polar_route/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.6.5"
+__version__ = "0.6.6"
```

Comment:
This needs to now change to `1.0.0`

Reply:
Refresh the page and check again? I updated this yesterday afternoon...